File: Tareas/tarea9.rb
puts "Azul Carolina Martínez Cabañas"
puts "1868435					 ITS"
puts "Periodo Enero-Junio 2020"
puts "Ing. Ismael Gutiérrez Pimentel"
puts "Convertidor de pesos mexicanos a divisas extranjeras.\nIngrese una de las siguientes opciones: "
puts "1 para dólar"
puts "2 para euro"
puts "3 para yen japonés"
puts "4 para dólar de Hong Kong"
puts "\n"
mon = gets().to_i

case mon
    when 1
        puts "Ingrese la cantidad de pesos que desea convertir: "
        n = gets().to_i
        dol = n / 20
        if dol > 1
            puts "#{dol} dólares"
        else
            puts "#{dol} dólar"
        end

    when 2
        puts "Ingrese la cantidad de pesos que desea convertir: "
        n = gets().to_i
        eur = n/23
        if eur > 1
            puts "#{eur} euros"
        else
            puts "#{eur} euro"
        end

    when 3
        puts "Ingrese la cantidad de pesos que desea convertir: "
        n = gets().to_i
        yen = n/0.18
        if yen > 1
            puts "#{yen} yenes"
        else
            puts "#{yen} yenes"
        end

    when 4
        puts "Ingrese la cantidad de pesos que desea convertir: "
        n = gets().to_i
        hk = n / 2.58
        if hk > 1
            puts "#{hk} dólares de Hong Kong"
        else
            puts "#{hk} dólar de Hong Kong"
        end
    else 
        puts "Ingresar una opción válida"
end

File: Tareas/tarea9.java
import javax.swing.*;

public class Tarea9 {
    public static void main(String[] args) {
        JOptionPane.showMessageDialog(null, "Azul Carolina Martínez Cabañas\n1868435 ITS\nPeriodo Enero-Junio 2020\nIng. Ismael Gutiérrez Pimentel");
        Integer mon = Integer.parseInt(JOptionPane.showInputDialog("\"Convertidor de pesos mexicanos a divisas extranjeras.\\nIngrese una de las siguientes opciones:\n" +
                "1 para dólar\n"+"2 para euro\n"+"3 para yen japonés\n"+"4 para dólar de Hong Kong"));
        switch(mon){
            case 1:
                Double pesDol = Double.parseDouble(JOptionPane.showInputDialog("Ingrese la cantidad de pesos que desea convertir: "));
                Double dol = pesDol / 20;
                JOptionPane.showMessageDialog(null, dol + " dólares");
                break;
            case 2:
                Double pesEur = Double.parseDouble(JOptionPane.showInputDialog("Ingrese la cantidad de pesos que desea convertir: "));
                Double eur = pesEur / 23;
                JOptionPane.showMessageDialog(null, eur + " euros");
                break;
            case 3:
                Double pesYen = Double.parseDouble(JOptionPane.showInputDialog("Ingrese la cantidad de pesos que desea convertir: "));
                Double yen = pesYen / 0.18;
                JOptionPane.showMessageDialog(null, yen + " yenes");
                break;
            case 4:
                Double pesHK = Double.parseDouble(JOptionPane.showInputDialog("Ingrese la cantidad de pesos que desea convertir: "));
                Double hk = pesHK / 2.58;
                JOptionPane.showMessageDialog(null, hk + " dólares de Hong Kong");
                break;
            default:
                JOptionPane.showMessageDialog(null, "Ingresar una opción valida");


        }
    }
}